src/main/java/Problem3/MovieAction.java

```diff
@@ -22,5 +34,12 @@ public int getLateFeeInDollar() {
     @Override
     public int calcLateFees(int numOfDaysPastDue) {
         // homework
+        if(numOfDaysPastDue > 0 && numOfDaysPastDue < 5){
+            return (numOfDaysPastDue * lateFeePerDayInDollar);
+        } else if(numOfDaysPastDue >= 5){
+            return (2 * numOfDaysPastDue * lateFeePerDayInDollar);
+        } else{
```

Comment:
"else" is not needed

Reply:
duplicated logic here. 

suggesting:
```java
        int fee = lateFeePerDayInDollar * numOfDaysPastDue;
        return (numOfDaysPastDue >= 5) ? 2 * fee : fee;
```

and here number 5 only appears once too